app/models/content_item.rb

```diff
@@ -52,6 +53,10 @@ def self.create_or_replace(base_path, attributes)
   validate :no_extra_route_keys
   validate :links_are_valid
   validate :access_limited_is_valid
+  validates :phase,
+            inclusion: { in: ['alpha', 'beta'],
+                         allow_blank: true,
```

Comment:
We should probably use `allow_nil` here, otherwise this will also accept an empty string.